circle.yaml

```diff
@@ -0,0 +1,8 @@
+deployment:
+  production:
+    branch: master
+    commands:
+      - node ./buildImageList.js
+      - git add .
```

Comment:
I would make this line explicit incase circle makes some other files. `git add imageList.json` should add only this file.